### ISSUE TITLE
Require Interaction to Dismiss Desktop Notification do not work in RC.Electron

### ISSUE THREAD

**Issue by ankar84** [RocketChat/Rocket.Chat.Electron#1505]
## My Setup
- Operating System: Windows 10
- App Version: 2.17.5
- Installation type: exe

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
I enabled **Require Interaction to Dismiss Desktop Notification** option in my settings.
When I test desktop notification in EC.Electron it shows **regular notification** (without Close button)
When I test desktop notification in Google Chrome I get notification, that **Require Interaction** (Close button)

### Current Behavior
RC.Electon
![image](https://user-images.githubusercontent.com/4023037/74000655-ecbe8200-4979-11ea-9c10-69ae35fdebcd.png)
Chrome:
![image](https://user-images.githubusercontent.com/4023037/74000630-d7e1ee80-4979-11ea-8680-e83df900db0b.png)

Now I logged in in both Electron and web version and here what I get:
![image](https://user-images.githubusercontent.com/4023037/74000687-11b2f500-497a-11ea-8978-576b571edef2.png)

### Expected Behavior
Get notification, that Require Interaction (Close button) in RC.Electron

**Comment by tassoevan** [comment]
I'm postponing this issue because Electron doesn't support it yet.

**Comment by ankar84** [comment]
In Version 3.0.1 notifications still go away despite **Require Interaction to Dismiss Desktop Notification**
![image](https://user-images.githubusercontent.com/4023037/94920244-20898180-04e0-11eb-9576-2260fcc53ce7.png)
![image](https://user-images.githubusercontent.com/4023037/94920267-2da67080-04e0-11eb-926a-27c75e1eca73.png)

**Comment by ankar84** [comment]
> I'm postponing this issue because Electron doesn't support it yet.

@tassoevan does Electron in 3.x versions now support that feature? 
Tested it 3.0.5 - not working.

**Comment by ankar84** [comment]
Still not working in new 3.4.0 on Chrome 91 (Electron 13 I guess)

**Comment by ankar84** [comment]
Still not working in 3.7.0 pre release on Electron 16.0.2

**Comment by ankar84** [comment]
Still an issue on recent 3.7.5

**Comment by jeanfbrito** [comment]
I will take a look if electron is supporting this now.

**Comment by ankar84** [comment]
Still not implemented in 3.7.8

**Comment by ankar84** [comment]
Still not implemented in 3.7.9

**Comment by ankar84** [comment]
Still not implemented in 3.8.8

**Comment by jeanfbrito** [comment]
And will not be implemented for some time. Windows don't gives this natively. 
I'm looking for some alternative, but for now it will not work yet.

**Comment by jeanfbrito** [comment]
Yes, the problem is that windows don't have this feature with electron. But
I'm working on a piece of concept that if work we will have very good
notifications on windows and this kind of one will be possible.

Em qua., 28 de set. de 2022 04:23, ankar84 ***@***.***>
escreveu:
